assets/src/edit-story/app/story/actions/useDeleteCurrentSelection.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * WordPress dependencies
+ */
+import { useCallback } from '@wordpress/element';
+
+// Delete the elements currently selected.
+function useDeleteCurrentSelection( {
```

Comment:
`useDeleteCurrentSelectedElement` ?

Reply:
That's actually for deleting the full selection, even if there's multiple elements selected. That's why the general name of `deleteCurrentSelection`

Reply:
Some jsdocs would be good here too.

Reply:
It is not clear from the name what is being deleted.

Reply:
Changed to `useDeleteSelectedElements`, this matches with the `selectedElements` state, too.